### PR TITLE
docs: add notice and rebrand

### PR DESCRIPTION
**Pull request by matt-FFFFFF**

### CODE REVIEW

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -138,7 +138,7 @@ const config = {
               // },
               {
                 label: 'GitHub',
-                href: 'https://github.com/azure/caf-terraform-landingzones',
```
Comment (LaurentLesle) [inline website/docusaurus.config.js:141]: See comment earlier on the name

File: website/versioned_docs/version-2203.1/azure-landing-zones/landingzones/platform/files_reference.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ The root of this file is called ```caf_terraform:``` and contains the following
 | azure_regions                     | yes      | map (key=region key, value= region)   | List of the Azure regions where services are authorized to be deployed. The region must be the small, lower-case with no space format (i.e: southeastasia)          |
 | prefix                            | yes      | string                                | Use a prefix to put in front of all names generated.                                                                                                                |
 | deployment_mode                   | yes      | 'platform' or 'asvm'                  | Inform the rover ignite if the deployments are for the platform or for an application landing zone.                                                                 |
-| caf_landingzone_branch            | yes      | tag or branch name                    | Release number from the repo https://github.com/Azure/caf-terraform-landingzones/releases                                                                           |
```
Comment (LaurentLesle) [inline website/versioned_docs/version-2203.1/azure-landing-zones/landingzones/platform/files_reference.md:27]: See comments on name

**Review by LaurentLesle — CHANGES_REQUESTED** [review]
See my comments

File: website/docs/azure-landing-zones/landingzones/platform/org-setup.mdx [hunk 3]
```diff
@@ -22,7 +22,7 @@ The video below shows you how to setup a private GitHub repository you are going
 
 The platform starter project is an empty environment that get you started with your initial configuration files and create a coherent stack.
 
-The [platform starter project is here](https://github.com/Azure/caf-terraform-landingzones-platform-starter).
```
[the remark this answers is not in the document]
Comment (LaurentLesle) [reply website/docs/azure-landing-zones/landingzones/platform/org-setup.mdx:25]: archived already

**Comment by arnaudlh** [comment]
Closing as changes already implemented.

**Pull request closed without merging**